.github/workflows/backend-test.yml

```
@@ -0,0 +1,32 @@
+name: Backend Tests
+
+on:
+  push:
+    paths:
+      - 'packages/backend/**'
+      - '.github/workflows/backend-test.yml'
+      - 'package.json'
+      - 'package-lock.json'
```

```suggestion
      - 'package.json'
      - 'pnpm-lock.yaml'
```
lock ファイルを pnpm に変更する必要がありそう